### ISSUE TITLE
Implementation of Date Http header is not compliant with the Specification

### ISSUE THREAD

**Issue by sri-miriyala** [mojaloop/project#451]
Priority : HIGHEST

When I sent a transfers-prepare request with the valid date filed in Headers as per the Mojaloop Spec, Im getting a response of "BadRequestError"
Attached is the screen shot:
<img width="781" alt="screen shot 2018-09-06 at 2 28 02 pm" src="https://user-images.githubusercontent.com/38143240/45180709-28d03f00-b1ea-11e8-8fda-54475379cc63.png">

Below is the API Spec screen shot :

<img width="1042" alt="screen shot 2018-09-06 at 2 40 26 pm" src="https://user-images.githubusercontent.com/38143240/45180729-37b6f180-b1ea-11e8-8df7-d4707fc2fe8f.png">
This issue is blocking clients to send transfers-prepare request to the switch.

** PRs
- [x] https://github.com/mojaloop/ml-api-adapter/pull/50

**Comment by sri-miriyala** [comment]
@elnyry Please do not close the defects opened by me as discussed before. I will close them once I re-test and validate that the bug is indeed fixed.

**Comment by elnyry** [comment]
Hi Sri, Sure from next time..

For stories - whenever the code is merged into the develop branch after reviews, we close the items. If the bug is not fixed, re-opening it is a valid action to do.

**Comment by elnyry** [comment]
If we just want to deal with these as bugs and not as stories/action-items, may be we can do that.

Also, FYI here is the format we adopted for Bugs - https://github.com/mojaloop/docs/blob/develop/Standards/Template_Bug.md

**Comment by sri-miriyala** [comment]
Yes, I prefer to deal with these as bugs and not stories. You can assign it back to me once the fix is in place, but I want to validate that the issue is fixed before closing the bug.

**Comment by mdebarros** [comment]
PR merged -> https://github.com/mojaloop/ml-api-adapter/pull/50

@elnyry

**Comment by mdebarros** [comment]
Waiting for new deployment Monday morning for @sri-miriyala to verify

**Comment by elnyry** [comment]
@sri-miriyala @mdebarros - I just made a new deployment that now includes the 3.5.2 snapshot version for ml-api-adapter that includes the Hotfix  for this.

**Comment by sri-miriyala** [comment]
It is working as expected and hence closed.
